src/manifest/representation.ts

```diff
@@ -111,6 +111,20 @@ class Representation {
    */
   public codec : string | undefined;
 
+  /**
+   * Supplemental codecs are defined as backwards-compatible codecs enhancing
```

Comment:
Why did you chose the `supplemental` and not the `additional` wording for this variable?

Reply:
Same wording on other specs like HLS.

There's also a subtle difference in my view between a supplement and an addition in both english (and french we 3 speak :P), meaning kind of "on top of" which is how I understand this property.

It's not just an addition, it's here a more precize codec string for devices that can handle it.